xmppserver/src/main/java/org/jivesoftware/openfire/muc/spi/LocalMUCRoom.java

```diff
@@ -1083,6 +1083,10 @@ void addOccupantRole( @Nonnull MUCRole role )
      * @param leaveRole the role of the occupant that is leaving.
      */
     CompletableFuture<Void> sendLeavePresenceToExistingOccupants(MUCRole leaveRole) {
+        // Do not send the presence for occupants on other nodes
+        if (!leaveRole.isLocal() || leaveRole.isRemoteFmuc())
```

Comment:
This is picky style thing, I know, but can we have braces around the statement for this if() statement please? It may help prevent bugs in the future as this code changes.

Reply:
changed